Backend/controllers/PatientsController.js

```diff
@@ -1,13 +1,14 @@
 const { Patient } = require("../models/Patient.js");
 var mongoose = require("mongoose");
+const { PatientLog } = require("../models/PatientLog.js");
 
 // getting all patients
 exports.get_all_patients = async (req, res) => {
     let filter = {};
 
-    if(req.query.user_id){
-        filter.user_id = req.query.user_id;
-    }
+    // if(req.query.user_id){
```

Comment:
Are we sure web team isn't using this? Last I checked mobile wasn't

Reply:
they are but since we are going to remove the users from the patients model..... that is why i removed it :)

Reply:
They are informed right? If so, just click resolve